src/authentication.hooks.js

```diff
@@ -46,9 +88,38 @@ module.exports = {
         try {
           if (context.data.strategy === 'local') {
             const db = await context.app.get('mongoClient');
-            const countUser = await db.collection('users').countDocuments({ name: context.data.name, resetPasswordCnil: true });
-            if (countUser > 0) {
+            const user = await db.collection('users').findOne({ name: context.data.name });
+            if (user?.resetPasswordCnil === true) {
               context.error = new Forbidden('RESET_PASSWORD_CNIL', { resetPasswordCnil: true });
+              return;
+            }
+            let attemptFail = user?.attemptFail ?? 0;
```

Comment:
ici on serait donc dans un if ERROR_ATTEMPT_LOGIN ?

Reply:
On est dans un context.error.className === 'not-authenticated' du coup, et j'ajoute Error: PROCESS_LOGIN_UNBLOCKED pour gérer le cas qui permet d'ouvrir la popin

Reply:
ah yes top